### ISSUE TITLE
Create a new Eval plan

### ISSUE THREAD

**Issue by lesleychou** [lesleychou/NetPress_benchmark#29]
Step:
1. Agents to be state-of-the-art: for zero-shot
2. Few-shot and Fine-tuning
	1. Can it solve task-1
	2. Can it solve task-10?
	3. Will the model overfit to another tasks

We observe fundamental issues

Generate fine-tuning example for both tasks, 
	train multiple tasks for the same model

Test static and AutoGen benchmark

OOD:
Static test set  VS Dynamic test set

Is there a fundamental insight? How do we adapt to domain-specific tasks

Relationship between task difficulty level and sample size? 
Level-1: need 10 samples
Level-2: may need 100 samples?

Some rules behind I/O
If you use RL to learn the rules behind I/O set
Let the model learn RL by itself

Q1: task are different application?
Q2: Dynamic set is OOD testing?
Q3: need X samples meaning for fine-tuning

## Experiments importance
1. Static VS Dynamic details
